packages/eslint-plugin/src/rules/ban-context-export/index.test.js

```diff
@@ -3,43 +3,40 @@ const { RuleTester } = require('@typescript-eslint/rule-tester');
 const path = require('path');
 const rule = require('./index');
 
-const ruleTester = new RuleTester({
```

Comment:
seems to redundant, worked before upgrade, but fails now, after removal there is no error and test are passing